control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -1182,6 +1182,10 @@ func (r *reconciler) reconcileAuthOIDC(ctx context.Context, hcp *hyperv1.HostedC
 		// Copy OIDCClient Secrets into openshift-config namespace
 		if len(hcp.Spec.Configuration.Authentication.OIDCProviders[0].OIDCClients) > 0 {
 			for _, oidcClient := range hcp.Spec.Configuration.Authentication.OIDCProviders[0].OIDCClients {
+				// If the secret name is empty, we assume the guest cluster admin is going to create this secret manually
```

Comment:
At this point do we know that `len(hcp.Spec.Configuration.Authentication.OIDCProviders) > 0`? Just to avoid panics

Reply:
The range will immediately exit if the len is 0 so the check is not needed.  I probably meant to check that `len(hcp.Spec.Configuration.Authentication.OIDCProviders) > 0`